orangecontrib/spectroscopy/tests/test_readers.py

```diff
@@ -70,6 +72,37 @@ def test_roundtrip(self):
             np.testing.assert_equal(d1.X, d2.X)
 
 
+class TestOpusReader(unittest.TestCase):
```

Comment:
Watch out with OPUS files, always add  `@unittest.skipIf(opusFC is None, "opusFC module not installed")` and also test without opusFC installed, because OpusFC is an optional library.

That is another reason to remove opus files from Hyper tests.

Reply:
Agree that it is unnecessary to use opus files to test HyperSpectra. Since I store image bytes (jpeg or bmp) in attributes it is a little bit difficult to come up with dummy image bytes.

I can mock `Image.open` everywhere as it done in `test_select_another_visible_image`. Do you think it is the way to do it?

Reply:
Well, I'd prefer not to mock it Image.open, I would rather mock the data. You could perhaps add a small image to the repo and then read that one as bytes. Or, if the image is really small (few pixels would be enough anyway), you could base64 encode it and just leave it in the test .py file.